components/TabView.js

```diff
@@ -43,7 +43,7 @@ const Div = s.div`
 const Tabs = s.div``
 
 const TabView = ({ tabs, tabClassName, background }) => {
-  const hashString = window.location.hash.substring(1)
+  const hashString = process.browser ? window.location.hash.substring(1) : null
```

Comment:
maybe put a comment about what this does? also lmk if this applies: https://github.com/zeit/next.js/issues/5354#issuecomment-426157764 

I'm most used to seeing `typeof window === 'undefined'`

Reply:
You're right, `typeof window === 'undefined'` also seems cleaner to me. Changed and left a comment!